secretsstoresynccontroller/config/admissionpolicies/create_secret_sync_object_exists_for_kubernetes_secret.yaml

```diff
@@ -0,0 +1,31 @@
+# This policy is used to prevent the creation of a Kubernetes secret if a SecretSync object doesn't exist for it.
+# TODO: Use parameters to allow the user to specify the name of the SecretSyncController.
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "secretsynccontroller-create-ownership-policy"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   [""]
+      apiVersions: ["v1"]
+      operations:  ["CREATE"]
+      resources:   ["secrets"]
+  variables:
+  - name: hasOneSecretSyncOwner
+    expression: "(size(object.metadata.ownerReferences) == 1 && object.metadata.ownerReferences.all(o, o.kind == 'SecretSync' && o.name == object.metadata.name))"
+  - name: userIsSecretSyncController
```

Comment:
move the userIsSecretSyncController check to a matchCondition so we skip everything else if the request isn't from a secret sync controller